src/add-and-remove.js

```diff
@@ -0,0 +1,24 @@
+const TODO = 'todo';
```

Comment:
## GOOD JOB SO FAR 👏

- [ ] There are parts that are missing in the `readme` file. Please kindly add the screenshot of the app and the MIT license ✅